### PR TITLE
Logging for Oracle and MySQL

### PR DESCRIPTION
**Pull request by rahulb-yb**

### CODE REVIEW

File: yb_migrate/src/migration/oracle.go [hunk 1]
```diff
@@ -282,15 +279,15 @@ func getSourceDSN(source *utils.Source) string {
 		os.Exit(1)
 	}
 
+	log.Infof("Source DSN used for export:%s\n", sourceDSN)
 	return sourceDSN
 }
 
 func OracleGetAllTableNames(source *utils.Source) []string {
 	dbConnStr := GetDriverConnStr(source)
```
Comment (sanyamsinghal) [inline yb_migrate/src/migration/oracle.go:287]: We can log the golang connection string here and other similar places.

File: yb_migrate/src/migration/oracle.go [hunk 2]
```diff
@@ -341,13 +336,13 @@ func OracleGetAllPartitionNames(source *utils.Source, tableName string) []string
...
-			errMsg := fmt.Sprintf("error in scanning query rows: %v\n", err)
-			utils.ErrExit(errMsg)
+			utils.ErrExit("error in scanning query rows: %v", err)
 		}
 		partitionNames = append(partitionNames, partitionName)
 
 		// TODO: Support subpartition(find subparititions for each partition)
 	}
 
+	log.Infof("Partition Names:%q\n", partitionNames)
```
Comment (sanyamsinghal) [inline yb_migrate/src/migration/oracle.go:346]: Log parent table name also with the partition name.
Something like this:
```
log.Infof("Partition Names for parent table %q: %q", partitionNames)
```

Aslo, I think "\n" with log.Infof() and log.Info() are not required. Please check and remove if the case

File: yb_migrate/src/migration/oracle.go [hunk 3]
```diff
@@ -93,29 +96,27 @@ func Ora2PgExtractSchema(source *utils.Source, exportDir string) {
...
 					runtime.Goexit()
 				}
 			}
 		}()
 
-		// fmt.Printf("[Debug] exportSchemaObjectCommand: %s\n", exportSchemaObjectCommand.String())
 		err := exportSchemaObjectCommand.Start()
 		if err != nil {
-			fmt.Println(err.Error())
+			utils.PrintAndLog("Error while starting export:%v", err)
```
Comment (sanyamsinghal) [inline yb_migrate/src/migration/oracle.go:107]: nit: space better ":" and "%v"

Comment (sanyamsinghal) [reply yb_migrate/src/migration/oracle.go:107]: similarly for other places.

File: yb_migrate/src/migration/oracle.go [hunk 4]
```diff
@@ -206,17 +205,17 @@ func Ora2PgExportDataOffline(ctx context.Context, source *utils.Source, exportDi
 
 	//Exporting all the tables in the schema
 	exportDataCommand := exec.Command("/bin/bash", "-c", exportDataCommandString)
-	// log.Debugf("exportDataCommand: %s", exportDataCommandString)
+	log.Infof("Executing command:%s\n", exportDataCommandString)
```
Comment (sanyamsinghal) [inline yb_migrate/src/migration/oracle.go:208]: space

File: yb_migrate/src/migration/oracle.go [hunk 5]
```diff
@@ -299,30 +296,29 @@ func OracleGetAllTableNames(source *utils.Source) []string {
...
 		if err != nil {
-			errMsg := fmt.Sprintf("error in scanning query rows for table names: %v\n", err)
-			utils.ErrExit(errMsg)
+			utils.ErrExit("error in scanning query rows for table names: %v", err)
 		}
 
 		tableNames = append(tableNames, tableName)
 	}
 
+	log.Infof("Table Name List:%q\n", tableNames)
```
Comment (sanyamsinghal) [inline yb_migrate/src/migration/oracle.go:312]: space

File: yb_migrate/src/migration/oracle.go (hunk 2 above)
Comment (rahulb-yb) [reply yb_migrate/src/migration/oracle.go:346]: There is an explicit log.Infoln() which appends "\n", the helper functions that we have written (utils.PrintandLog(), utils.ErrExit()) have an implicit newline appended

File: yb_migrate/src/migration/oracle.go (hunk 1 above)
Comment (rahulb-yb) [reply yb_migrate/src/migration/oracle.go:287]: This will be handled in the logging for common.go, that is where the connection string is formed

**Review by sanyamsinghal — APPROVED** [review]

**Pull request merged**